.github/workflows/e2e.yaml

```diff
@@ -48,7 +48,24 @@ jobs:
 
       - name: Test Policy
         run: |
-          make test-cmd
+          if [[ ${{ github.event_name }} == "push" ]]
+          then
+            GIT_BRANCH=${GITHUB_REF##*/}
+          elif [[ ${{ github.event_name }} == "pull_request" ]]
+          then
+            GIT_BRANCH=${{ github.event.pull_request.base.ref }}
```

Comment:
Looks good now!

One more request, can you add release branches to `e2e.yaml` . For example, on `release-1.5`, e2e tests are also triggered on the release branch:
 https://github.com/kyverno/kyverno/blob/a4cd3a571f2a60e4ca4c5a36035dd34852fe0076/.github/workflows/e2e.yaml#L6
https://github.com/kyverno/kyverno/blob/a4cd3a571f2a60e4ca4c5a36035dd34852fe0076/.github/workflows/e2e.yaml#L14

Reply:
> One more request, can you add release branches to `e2e.yaml` .

Sure